library/src/main/java/com/schibsted/spain/barista/rule/flaky/FlakyTestRule.kt

```diff
@@ -29,10 +29,23 @@ class FlakyTestRule : TestRule {
     return this
   }
 
+  /**
+   * Utility method to use @[Repeat] by default in all test methods.
+   * <br></br>
+   * Use this method when constructing the Rule to apply a default behavior of @[Repeat] without having to add the annotation to
+   * each test. This can help you to find flaky tests.
+   * <br></br>
+   * The default behavior can be overridden with [Repeat] or [Repeat].
```

Comment:
`[Repeat] or [Repeat]`?

Reply:
It was `[Repeat] or [AllowFlaky]`, fixed!